test/auth.test.js

```diff
@@ -55,6 +55,16 @@ describe('support for HTTP Authentication', function() {
        fails('/digestAuth'));
   });
 
+  describe('remotes.auth', function () {
+    it('should be populated from the url', function () {
+      var credentials = 'foo:bar';
+      var url = fmt('http://%s@myhost.com', credentials);
```

Comment:
Is there any particular reason for using `fmt` instead of including the credentials in the url string? Also `foo:bar` is not very descriptive,

Proposal:

```js
var url = 'http://login:pass@myhost.com';
```